test/integration/instance/update.test.js

```diff
@@ -60,19 +60,19 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
     });
 
     if (current.dialect.supports.transactions) {
-      it('supports transactions', function(done) {
+      it('supports transactions', function() {
         Support.prepareTransactionTest(this.sequelize, function(sequelize) {
```

Comment:
`return Support.prepareTransactionTest(this.sequelize).then(function (sequelize) {`